src/components/transaction-address.tsx

```diff
@@ -81,28 +86,35 @@ export const TransactionAddress = ({
 
   return (
     <KeyboardSafeArea>
-      <TextField
-        label="Send to"
-        style={page.input}
-        placeholder="Enter Address or contact name"
-        value={address}
-        onChangeText={setAddress}
-        error={error}
-        errorText="Incorrect address"
-        autoFocus
-        multiline
-        rightAction={
-          address === '' ? (
-            <IconButton onPress={onPressQR}>
-              <QRScanner color={GRAPHIC_GREEN_1} width={25} height={25} />
-            </IconButton>
-          ) : (
-            <IconButton onPress={onPressClear}>
-              <CloseCircle color={GRAPHIC_BASE_2} width={25} height={25} />
-            </IconButton>
-          )
-        }
-      />
+      <View>
```

Comment:
that the reason for this view?

Reply:
this helps to correctly position the absolute position of the placeholder

Reply:
ok